packages/client/components/ReflectionGroup/ReflectionGroup.tsx

```diff
@@ -171,6 +192,7 @@ const ReflectionGroup = (props: Props) => {
         {...(isBehindSpotlight ? null : {[DragAttribute.DROPPABLE]: reflectionGroupId})}
         ref={groupRef}
         staticReflectionCount={staticReflections.length}
+        isSpotlightSource={isSpotlightSrcGroup && !isBehindSpotlight}
```

Comment:
We use the `staticReflectionCount` to determine the `bottomPadding` on [L35](https://github.com/ParabolInc/parabol/blob/a20a10d19b23c739a823a07488586e9aede6f0aa/packages/client/components/ReflectionGroup/ReflectionGroup.tsx#L35). The Spotlight clone doesn't know what the `staticReflectionCount` is, resulting in the clone & source having different padding (only when `spotlightGroup` is a stack of reflections). A few options:
1. Set the `staticReflectionCount` to 1 if it's the source group. The source doesn't need extra padding.
2. Pass an `isSpotlightSource` prop & use a ternary in the `paddingBottom`. 
3. Calculate the `staticReflectionCount` in `useAnimatedSpotlightSource` and add the `paddingBottom` the clone. This is tricky as we need the `visibleReflections` (and therefore `hiddenIds`) to calculate the `staticReflection`.

I've opted for option 2 as it's simpler than 3 and more explicit than 1.